source/speech/commands.py

```diff
@@ -229,21 +239,41 @@ def __repr__(self):
 		return "{type}({param})".format(
 			type=type(self).__name__, param=param)
 
+
 class PitchCommand(BaseProsodyCommand):
 	"""Change the pitch of the voice.
 	"""
 	settingName = "pitch"
 
+
 class VolumeCommand(BaseProsodyCommand):
 	"""Change the volume of the voice.
 	"""
 	settingName = "volume"
 
+	def __init__(self, offset=0, multiplier=1):
+		if not (offset == 0 and multiplier == 1):
+			log.warning(
+				f'If defaultVal of Rate attribute is 0, calling the multiplier property '
+				f'will result in a ZeroDivisionError.'
+			)
+		super().__init__(offset, multiplier)
```

Comment:
I don't this warning is very helpful at the moment. The condition should also check if defaultVal is 0. Better yet would be to consider the implications of taking the same approach for Volume and Rate.